tools_docker/common.sh

```diff
@@ -1,24 +1,28 @@
     CONFIG_FILE_IS_NEW=1
-    cp /tracim/backend/development.ini.sample /etc/tracim/config.ini
-    sed -i "s|listen = .*|listen = 127.0.0.1:6543|g" /etc/tracim/config.ini
-    sed -i "s/\(depot_storage_dir *= *\).*/depot_storage_dir = \/var\/tracim\/depot/" /etc/tracim/config.ini
-    sed -i "s|\(session.data_dir *= *\).*|session.data_dir = \/var\/tracim\/session.data\/|g" /etc/tracim/config.ini
-    sed -i "s|\(session.lock_dir *= *\).*|session.lock_dir = \/var\/tracim\/session.lock\/|g" /etc/tracim/config.ini
+    KEY=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w ${1:-32} | head -n 1)
+    SECRET=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w ${1:-32} | head -n 1)
+    cp /tracim/backend/development.ini.sample /etc/tracim/production.ini
+    sed -i "s|basic_setup.api_key =.*|basic_setup.api_key = $KEY|g" /etc/tracim/production.ini
+    sed -i "s|basic_setup.session_secret = change_this_value_please\!|basic_setup.session_secret = $SECRET|g" /etc/tracim/production.ini
+    sed -i "s|basic_setup.website_base_url = .*|basic_setup.website_base_url = http://localhost:8080|g" /etc/tracim/production.ini
+    sed -i "s|basic_setup.listen = .*|basic_setup.listen = 127.0.0.1:8080|g" /etc/tracim/production.ini
+    sed -i "s|basic_setup.depot_storage_dir = .*|basic_setup.depot_storage_dir = \/var\/tracim\/depot|g" /etc/tracim/production.ini
+    sed -i "s|basic_setup.sessions_data_root_dir = .*|basic_setup.sessions_data_root_dir = \/var\/tracim|g" /etc/tracim/production.ini
     case "$DATABASE_TYPE" in
       mysql)
-        sed -i "s/\(^sqlalchemy.url *= *\).*/\\sqlalchemy.url = $DATABASE_TYPE+pymysql:\/\/$DATABASE_USER:$DATABASE_PASSWORD@$DATABASE_HOST:$DATABASE_PORT\/$DATABASE_NAME$DATABASE_SUFFIX/" /etc/tracim/config.ini ;;
+        sed -i "s|basic_setup.sqlalchemy_url = .*|basic_setup.sqlalchemy_url = $DATABASE_TYPE+pymysql:\/\/$DATABASE_USER:$DATABASE_PASSWORD@$DATABASE_HOST:$DATABASE_PORT\/$DATABASE_NAME$DATABASE_SUFFIX|g" /etc/tracim/production.ini ;;
       postgresql)
-        sed -i "s/\(^sqlalchemy.url *= *\).*/\\sqlalchemy.url = $DATABASE_TYPE:\/\/$DATABASE_USER:$DATABASE_PASSWORD@$DATABASE_HOST:$DATABASE_PORT\/$DATABASE_NAME$DATABASE_SUFFIX/" /etc/tracim/config.ini ;;
+        sed -i "s|basic_setup.sqlalchemy_url = .*|basic_setup.sqlalchemy_url = $DATABASE_TYPE:\/\/$DATABASE_USER:$DATABASE_PASSWORD@$DATABASE_HOST:$DATABASE_PORT\/$DATABASE_NAME$DATABASE_SUFFIX|g" /etc/tracim/production.ini ;;
       sqlite)
-        sed -i "s/\(^sqlalchemy.url *= *\).*/\\sqlalchemy.url = sqlite:\/\/\/\/var\/tracim\/tracim.sqlite/" /etc/tracim/config.ini ;;
+        sed -i "s|basic_setup.sqlalchemy_url = .*|basic_setup.sqlalchemy_url = sqlite:\/\/\/\/var\/tracim\/tracim.sqlite|g" /etc/tracim/production.ini ;;
     esac
 fi
 if [ ! -L /tracim/backend/development.ini ]; then
-    ln -sf /etc/tracim/config.ini /tracim/backend/development.ini
+    ln -sf /etc/tracim/production.ini /tracim/backend/development.ini
```

Comment:
you create a link called production.ini which link to development.ini. Its unclear. If its production, then both should be named production.ini, if its not production, both should be named development.ini

Reply:
its not possible to change all. I have tried this but its not working with backend code.
Backend code use name `development.ini` for this file, I can use correctly some command if I change this name in backend folder.

I have make this name for file in volume.

Reply:
After investigate we need to find good way to use generic name or to make a real possibility to change name of config file.

Name is used in script, in backend code and in file for build and start docker.

Reply:
After discuss, we save name development.ini for the moment.